frontend/src/components/Smart/index.jsx

```diff
@@ -102,15 +102,15 @@ class Smart extends React.Component {
 
         if (adminTabsAvailable) {
             adminTabRecycle = (
-                <Tab eventKey={5} transition={false} title={<span className="glyphicon glyphicon-trash" aria-hidden="true"></span>} className="full card">
+                <Tab eventKey={5} transition={false} title={<div><span className="glyphicon glyphicon-trash" aria-hidden="true"></span> Discarded Data</div>} className="full card">
                     <div className="cardContent">
                         <RecycleBinContainer />
                     </div>
                 </Tab>
             );
         } else {
             adminTabRecycle = (
-                <Tab eventKey={5} transition={false} title={<span className="glyphicon glyphicon-trash" aria-hidden="true"></span>} className="full card">
+                <Tab eventKey={5} transition={false} title={<div><span className="glyphicon glyphicon-trash" aria-hidden="true"></span> Discarded Data</div>} className="full card">
```

Comment:
I think the `span` could use some margin/padding on the left to give it some space from the trash can icon.